code-of-conduct.md

```diff
@@ -0,0 +1,147 @@
+ethnicity, religion, age, physical appearance, body size, race, or similar
+personal characteristics; inappropriate use of nudity, sexual images, and/or
+sexually explicit language in public spaces; threats of physical or non-
+physical harm; deliberate intimidation, stalking or following; harassing
+photography or recording; sustained disruption of talks or other events;
+inappropriate physical contact; and unwelcome sexual attention.
+
+Media shared through public infrastructure run by the Quill team must not
+contain illegal or infringing content. You should only publish content via Quill
+public infrastructure if you have the right to do so. This includes complying
+with all software license agreements or other intellectual property
+restrictions. You will be solely responsible for any violation of laws or
+others’ intellectual property rights.
+
+### Reporting Violations of this Code of Conduct
+If you believe someone is harassing you or has otherwise violated this Code of
+Conduct, please contact us at abuse@quill.is to send us an abuse report. If this
+is the initial report of a problem, please include as much detail as possible.
+It is easiest for us to address issues when we have more context.
+
+### Copyright Violations
+We respect the intellectual property of others and ask that you do too. If you
+believe any content or other materials available through public Quill
+infrastructure violates a copyright held by you and you would like to submit a
+notice pursuant to the Digital Millennium Copyright Act or other similar
+international law, you can submit a notice to our agent for service of notice
+to: abuse@quill.is
+
+(We will add a physical mailing address here when we acquire one).
+
+Please make sure your notice meets the Digital Millennium Copyright Act
+requirements.
+
+### Consequences
+All content published to public Quill infrastructure is hosted at the sole
+discretion of the Quill team.
+
+Unacceptable behavior from any community member will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the Quill team may take
+any action they deem appropriate, up to and including a temporary ban or
+permanent expulsion from the community without warning (and without refund in
+the case of a paid event or service).
+
+### Addressing Grievances
+If you feel you have been falsely or unfairly accused of violating this Code of
+Conduct, you should notify the Quill team. We will do our best to ensure that
+your grievance is handled appropriately.
+
+In general, we will choose the course of action that we judge as being most in
+the interest of fostering a safe and friendly community.
+
+On IRC, let one of the ops know if you think that someone has transgressed
+against the Code of Conduct.
+
+### Contact Info
+Please contact abuse@quill.is if you need to report a problem or address a
+grievance related to an abuse report.
+
```

Comment:
Does this include cases of the community policing itself? "Hey guys, I saw this and I'm pretty sure this is copyrighted/not allowed. Could you take a look at it?"